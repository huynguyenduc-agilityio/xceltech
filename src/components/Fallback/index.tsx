import { LoaderCircle } from 'lucide-react';

const Fallback = () => (
  <div className="flex h-full items-center justify-center">
    <LoaderCircle className="h-12 w-12 animate-spin text-gray-300" />
  </div>
);

export default Fallback;
