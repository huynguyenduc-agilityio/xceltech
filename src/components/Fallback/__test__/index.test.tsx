import { render } from '@testing-library/react';

// Component
import Fallback from '..';

describe('Fallback Component', () => {
  it('should match snapshot', () => {
    const element = render(<Fallback />);

    expect(element).toMatchSnapshot();
  });
});
