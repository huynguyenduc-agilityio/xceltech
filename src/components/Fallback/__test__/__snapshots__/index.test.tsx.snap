// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fallback Component should match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex h-full items-center justify-center"
      >
        <svg
          class="lucide lucide-loader-circle h-12 w-12 animate-spin text-gray-300"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M21 12a9 9 0 1 1-6.219-8.56"
          />
        </svg>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex h-full items-center justify-center"
    >
      <svg
        class="lucide lucide-loader-circle h-12 w-12 animate-spin text-gray-300"
        fill="none"
        height="24"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21 12a9 9 0 1 1-6.219-8.56"
        />
      </svg>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
